calibration/FLORTD/CGINS-FLORTD-01197__20200108.csv

```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes

```

Comment:
The orginal cal date in this dev file is incorrect. Corrected here from the factory PDF sheets.